### DISCUSSION TITLE
Allow hiding the user part of the device name or update based on current user

### DISCUSSION THREAD

**Discussion by andrewheberle** [rustdesk/rustdesk#7955] [Ideas]
When a RustDesk client is first added the device is named `username@device_name` however in an environment where different users may log onto a machine this information is no longer correct.

It may be possible to implement this in a few ways if it is useful:

- Show the device name the same as it is now
- Update the username based on the currently logged in user, with only the device name shown if no user is logged in
- Only show the device name
- Allow some configuration option to select the above...maybe via a Strategy?

**Comment by njc-paul** [comment]
Would be better to have two separate columns Username (current user) and device name, personally I often find myself searching for both for different reasons
